pkg/authorization/api/types.go

```diff
@@ -20,18 +20,53 @@ const (
 	VerbAll     = "*"
 )
 
+const (
+	ResourceGroupPrefix         = "resourcegroup"
```

Comment:
Would this be a good place to godoc what each group includes? The internal/exposed and granter groups are the most opaque, I think

On a related note, we should keep a list of all the things someone adding a new resource type needs to do, and put these things on that list:

n. Add to appropriate policy resource groups
n+1. Add to bootstrap policy roles as needed
n+2. Add to bootstrap policy bindings as needed